### DISCUSSION TITLE
[Fundamentos do C#] Dúvida de como visualizar as Classes e os Métodos no visual studio code

### DISCUSSION THREAD

**Discussion by Pudim-Enlatado** [balta-io/forum#668] [Tira Dúvidas] [answered]
Opa
Recentemente iniciei o curso dos Fundamentos do C#, e estou no módulo [Linguagem de Programação com C#], e me surgiu uma dúvida na aula de Métodos e Funções. Quando esse curso foi criado, ainda era a versão antiga do VS code, então era possível visualizar ao criar uma aplicação, a Classe e o Método dela.: 
![imagem_2024-08-21_201503837](https://github.com/user-attachments/assets/b0ddebe8-0f18-4430-a8c6-9d6d160bb3f3)
Porém, hoje em dia quando se cria uma aplicação, não aparece mais.:
![imagem_2024-08-21_202036646](https://github.com/user-attachments/assets/e8820991-6a7a-4233-bf5f-5daf317b7611)
Portando queria saber, como essa aula funcionária pra versão mais recente do Visual Studio Code, onde a timeline é mais "simplificada".

**Comment by henriqueweege** [comment]
Fala, @Pudim-Enlatado ! Tudo certinho?
Então, essa não é uma mudança na IDE, mas sim do .NET. A partir do .NET 6 não é mais necessário ter de maneira explícita a classe Program nem o método Main.
Quanto a como seguir a aula, se preferir, você pode colar o código abaixo direto no arquivo Program.cs e seguir normal. Mas não há necessidade, se você escrever o código que está na função Main na raiz do Program.cs, sem o top-level statements, vai funcionar também!

```csharp
namespace MyApp
{
    internal class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
        }
    }
}
```



Se quiser ler mais sobre, tem [esse artigo](https://learn.microsoft.com/en-us/dotnet/core/tutorials/top-level-templates), e [esse aqui tbm](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/program-structure/top-level-statements).

Valeeeu!

**Reply by Pudim-Enlatado** [reply]
vlw, ajudou demais

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado:
https://github.com/balta-io/forum/discussions/602
